uktrade/data-hub-frontend
# Fix/export wins form uk region

## Description of change

This PR changes the source of options to the `company_contacts` field in the `customer_details` step of the `ExportWins/Form` from `/v4/metadata/uk-region` to `/v4/metadata/winukregion`.

## Test instructions

1. Go to `/companies/<company-id>/exportwins/create?step=customer_details`
2. Verify that options for the _HQ location_ field are populated from the `/v4/metadata/winukregion` endpoint
